worlds/hk/docs/setup_es.md

```diff
@@ -0,0 +1,51 @@
+   * Si lo deseas, instala también el "Archipelago Map Mod" para tener un mapa y tracker dentro del juego.
+3. Ejecuta el juego, ¡Ya está todo listo!
+
+### Qué hacer si Lumafly falla al buscar tu instalación de Xbox Game Pass
+
+1. Entra a la APP de Xbox y pon tu mouse sobre "Hollow Knight" en la barra lateral izquierda.
+2. Haz clic derecho y luego haz clic en "Administrar".
+3. Ve a la pestaña de "Archivos" y luego selecciona "Navegar".
+4. Haz clic en "Hollow Knight", luego en "Content", y luego haz click en la barra de directorio y copia la dirección de la carpeta.
+5. Ejecuta Lumafly como administrador y cuando te pregunte por la dirección de la carpeta, pega lo que has copiado en el paso 4.
+
+#### Método alternativo:
+1. Haz clic en tu perfil y luego en "Settings".
+2. Ve a la pestaña de "General" y selecciona "CHANGE FOLDER".
+3. Busca una carpeta en la que quieras instalar el juego (Preferiblemente en alguna carpeta en tu escritorio) y copia la dirección.
+4. Ejecuta Lumafly como administrador y cuando te pregunte por la dirección de la carpeta, pega lo que has copiado en el paso 3.
+
+Nota: La carpeta necesaria para el funcionamiento tiene que tener la carpeta "Hollow Knight_Data" en su interior
+
+## Configurar tu archivo YAML
+### ¿Qué es un YAML y por qué necesito uno?
+Puedes ver la [Guía basica de instalación del Multiworld](/tutorial/Archipelago/setup/en) aquí en el sitio de Archipelago para aprender acerca de por qué Archipelago usa archivos YAML y para que son.
+
+### ¿Cómo consigo un YAML? 
+Puedes usar la [página de configuraciones para Hollow Knight](/games/Hollow%20Knight/player-settings) aquí en el sitio de Archipelago para generar un YAML usando una interfaz grafica.
+
+### Unirse a una partida de Archipelago en Hollow Knight
+1. Inicia el juego después de haber instalado todos los mods necesarios.
+2. Crea una **nueva partida**.
+3. Selecciona el modo de juego **Archipelago** en la pantalla de selección de modo.
+4. Ingresa los campos correspondientes al servidor de Archipelago que vas a jugar.
+5. Presiona **Start** para inciar la partida, el juego se detendrá por unos segundos mientras configura todos los posicionamientos de objetos.
+6. El juego inmediatamente se iniciará en el juego randomizado.
+   * Si estas esperando por una cuenta atras para empezar, espera aque termine antes de presionar Start.
+   * O presiona el botón de Pausa una vez estes dentro del juego.
+   
+## Comandos
+Mientras juegas en el Multiworld puedes interactuar con el servidor con una variedad de comandos anotados en la
+[Guía de comandos](/tutorial/Archipelago/commands/en). Como este juego no tiene actualmente un cliente de texto, puedes de forma opcional conectarte al multiworld con el cliente de Texto, que puede ser encontrado en la
+[Instalación principal de Archipelago](https://github.com/ArchipelagoMW/Archipelago/releases) como "Archipelago Text Client" para introducir
```

Comment:
This might be outdated in English also, but I think there is no longer "Archipelago Text Client" in the installer.

There is a "Text Client" button in the launcher.

Reply:
probably should just be changed to say you can find the Text Client in the Archipelago Launcher tbh